README.md

```diff
@@ -40,109 +116,222 @@ You can download the script from the following locations:
 
         wget https://github.com/sandain/MinecraftServerControlScript/archive/master.zip
 
-* Make a clone of the [git repository](https://github.com/sandain/MinecraftServerControlScript.git):
 
-        git clone https://github.com/sandain/MinecraftServerControlScript.git
 
 ### Configuration
-To get your server to run the script on startup, and cleanly down the server
-on shutdown, the `mscs` script must be copied to `/usr/local/bin/`,
-have its execute permissions set, and the system must be instructed to use
-the script on startup and shutdown.  For Bash programmable completion
-support, the `mscs.completion` script must be copied to
-`/etc/bash_completion.d/`.  For security reasons, the script uses a user
-account named `minecraft` rather than `root` and the account must be created
-before the script is used.
-
-This can all be done automatically with the included Makefile in Debian and
+
+Navigate to the `MinecraftServerControlScript` directory that you just downloaded. Configuration can be done with the included Makefile in Debian and
 Ubuntu like environments by running:
 
-    sudo make install
+        sudo make install
+Then, type
 
-You can manually add the `minecraft` user and install the script with the
-following commands:
+        chmod -R u+w /opt/mscs
+        chown -R minecraft:minecraft /opt/mscs
+This will give the user you created in the config (by default, the user `minecraft`) access to write in the `/opt/mscs` folder. If you configured MSCS manually when you installed the script, then replace `minecraft` with the name of the user you made.
 
-    sudo adduser --system --group --home /opt/mscs --quiet minecraft
-    sudo install -m 0755 msctl /usr/local/bin/msctl
-    sudo install -m 0755 mscs /usr/local/bin/mscs
+That's it!
+If you wish to configure the script manually, please visit the [wiki page](https://github.com/Roflicide/MinecraftServerControlScript/wiki/Manual-Configuration).
```

Comment:
Should go to https://github.com/sandain/MinecraftServerControlScript/wiki/Manual-Configuration